### ISSUE TITLE
File Source (Async.)::Output error only happenes when It play with 'Play next file in the folder' option

### ISSUE THREAD

**Issue by tony9959** [clsid2/mpc-hc#1588]
When I play the video files indivisually, it plays very well.
However, When I set the 'Play next file in the folder' option on and play the next file automatically, The player shows me a File Source (Async.)::Output error.

File Source (Async.)::Output

Media Type 0:
--------------------------
Unknown

AM_MEDIA_TYPE: 
majortype: MEDIATYPE_Stream {E436EB83-524F-11CE-9F53-0020AF0BA770}
subtype: MEDIASUBTYPE_MPEG1Audio {E436EB87-524F-11CE-9F53-0020AF0BA770}
formattype: TIME_FORMAT_NONE {00000000-0000-0000-0000-000000000000}
bFixedSizeSamples: 1
bTemporalCompression: 0
lSampleSize: 1
cbFormat: 0

Media Type 1:
--------------------------
Unknown

AM_MEDIA_TYPE: 
majortype: MEDIATYPE_Stream {E436EB83-524F-11CE-9F53-0020AF0BA770}
subtype: TIME_FORMAT_NONE {00000000-0000-0000-0000-000000000000}
formattype: TIME_FORMAT_NONE {00000000-0000-0000-0000-000000000000}
bFixedSizeSamples: 1
bTemporalCompression: 0
lSampleSize: 1
cbFormat: 0

Here is the full error log. Is there any ways to fix it?

**Comment by clsid2** [comment]
Does a specific file trigger this?

**Comment by tony9959** [comment]
> Does a specific file trigger this?

I thought It happenes on every files but only a few ril trigger this error.  The files I tested were both mkv files so I can't find the difference

**Comment by clsid2** [comment]
It seems to be opening another file type than mkv. There is probably a few other files in that folder besides mkv files.

**Comment by Atrillion** [comment]
I recreated this same problem yesterday. Let me describe how. I opened FREEMAKE which allows me to join movie files together. I joined three .mp4 files and the file it created gave me this error. I looked over the settings and it was set to encode using CUDA which my Graphic Card although an Nvidea, isn't able to handle. So I changed the settings to encode using DirectX and walla. It worked. Maybe your graphic card can't handle this CUDA encoded file and if you convert it using DirectX it might work again, possibly. Use FREEMAKE, it's a great conversion tool. I am not associated with FREEMAKE but only a customer. If you reject the lifetime subscription of $60 it will offer it for a deal of $30 instead.
